relationships/candidates/ENVOY.yml

```diff
@@ -0,0 +1,34 @@
+category: ENVOY
+lookups:
+  - entityTypes:
+      - domain: EXT
+        type: ENVOY
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: ["cluster_name"]
```

Comment:
Updated the predicates so that the field matches the field in the relationship synthesis rules.